templates/registration/login.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h2>Login</h2>
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+  <form action="{% url 'login' %}" method="post">
+  {% csrf_token %}
+  {{ form.as_p }}
+  <input type="submit" value="Submit">
+  <input type="hidden" name="next" value="{{ next }}">
```

Comment:
The use of the `next` hidden input field is correct for redirecting users to their intended destination after a successful login. Ensure that the `next` context variable is properly set in the view handling this form.